### PR TITLE
Markup layout

### PR DESCRIPTION
**Pull request by abukurov**

### CODE REVIEW

File: app/components/search-input/style.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
+& {
+
```
Comment (IgorZiegler) [inline app/components/search-input/style.scss]: Why should we add this empty line?

File: app/components/search-input/style.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+.search-input-icon,
+.search-input-control {
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.search-input-icon {
+  font-size: 1.875em;
+  margin-right: 0.425em;
+
```
Comment (IgorZiegler) [inline app/components/search-input/style.scss]: Why should we add this empty line?

File: app/components/search-input/template.hbs [hunk 3]
```diff
@@ -0,0 +1,2 @@
+{{fa-icon "search" class="search-input-icon"}}
+{{input class="search-input-control" placeholder="Поиск"}}
```
Comment (maxmalov) [inline app/components/search-input/template.hbs]: Input component has 2way bindings, which are out of DDAU approach

File: app/index.html [hunk 4]
```diff
@@ -11,6 +11,7 @@
 
     <link rel="stylesheet" href="{{rootURL}}assets/vendor.css">
     <link rel="stylesheet" href="{{rootURL}}assets/vp-dashboard-ember.css">
+    <link href='https://fonts.googleapis.com/css?family=Ubuntu:400,400italic,500italic,500,700&subset=latin,cyrillic' rel='stylesheet' type='text/css'>
```
Comment (maxmalov) [inline app/index.html]: I think there is an add-on for that

File: package.json [hunk 5]
```diff
@@ -37,28 +37,36 @@
...
+    "eslint-config-airbnb-base": "^3.0.1",
     "eslint-config-netguru-ember": "0.0.5",
     "eslint-plugin-import": "^1.12.0",
     "eslint-plugin-netguru-ember": "0.0.5",
     "husky": "^0.11.6",
     "loader.js": "^4.0.1",
     "rimraf": "^2.5.4"
+  },
+  "dependencies": {
+    "normalize-scss": "^5.0.3"
```
Comment (maxmalov) [inline package.json]: All does should be in devDependencies section. Production bundle has no dependencies

File: tests/unit/search/controller-test.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('controller:search', 'Unit | Controller | search', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
```
Comment (maxmalov) [inline tests/unit/search/controller-test.js]: Won't ember warn about it? Unit tests should have needs, or unit: true

File: app/components/nav-header/style.scss [hunk 7]
```diff
@@ -0,0 +1,55 @@
+& {
+  background-color: $nav-bg-color;
+  color: white;
+}
+
+.navbar-inner {
```
Comment (maxmalov) [inline app/components/nav-header/style.scss]: Note this class is going to be global. Shouldn't we put in root `& { }`?

File: package.json (hunk 5 above)
Comment (maxmalov) [inline package.json]: *deps

File: ember-cli-build.js [hunk 8]
```diff
@@ -4,7 +4,26 @@ var EmberApp = require('ember-cli/lib/broccoli/ember-app');
...
+    sourcemaps: {
+      enabled: true,
+      extensions: ['js']
+    },
+    sassOptions: {
+      sourceMap: true
+    },
+    hinting: false,
+    babel: {
+      includePolyfill: true
```
Comment (maxmalov) [inline ember-cli-build.js]: I guess we can include them later, when it will be needed

File: app/application/template.hbs [hunk 9]
```diff
@@ -0,0 +1,11 @@
+<div class="vp-app-inner-wrap">
```
Comment (maxmalov) [inline app/application/template.hbs]: Let's be consistent about inner elements, this has inner-wrap suffix, but navbar has only inner

Comment (maxmalov) [inline app/application/template.hbs]: And do we really need this wrapper?

File: app/components/nav-header/style.scss (hunk 7 above)
Comment (maxmalov) [inline app/components/nav-header/style.scss]: And why it is a navbar? I mean it supposed to be a header. Even root element is a `header` in html. No need to copy bootstrap if we don't want to use it ;)

File: tests/integration/components/navigation-group/component-test.js [hunk 10]
```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('navigation-group', 'Integration | Component | navigation-group', {
+  integration: true
+});
+
+test('should expand and collapse content by click', function (assert) {
+  assert.expect(3);
```
Comment (maxmalov) [inline tests/integration/components/navigation-group/component-test.js]: don't use assertion planing for such simple test

**Comment by maxmalov** [comment]
LGTM

**Pull request merged**